.github/workflows/release-build.yml

```diff
@@ -18,11 +18,11 @@ jobs:
           ref: main
       - name: Reclaim Disk Space
         run: .github/ci-prerequisites.sh
-      - name: Set up JDK 8
+      - name: Set up JDK 1
```

Comment:
Maybe a typo here? You meant 11 and not 1?

Reply:
Ah good catch. Thanks!